shell/common/fixtures/shell_test.dart

```diff
@@ -482,3 +482,42 @@ Future<void> testThatAssetLoadingHappensOnWorkerThread() async {
   } catch (err) { /* Do nothing */ }
   notifyNative();
 }
+
+@pragma('vm:external-name', 'NativeReportViewIdsCallback')
+external void nativeReportViewIdsCallback(bool hasImplicitView, List<int> viewIds);
+
+List<int> getCurrentViewIds() {
+  return PlatformDispatcher.instance.views
+      .map((FlutterView view) => view.viewId)
+      .toSet()
```

Comment:
Why the `.toSet()`? It would be unexpected if there's repeated view ID values.

Reply:
I've changed it to an assertion that there are no duplicate view ID.